src/olympia/devhub/views.py

```diff
@@ -1145,30 +1144,6 @@ def check_validation_override(request, form, addon, version):
         helper.actions['super']['method']()
 
 
-def auto_sign_file(file_):
-    """If the file should be automatically reviewed and signed, do it."""
-    addon = file_.version.addon
-
-    if file_.is_experiment:  # See bug 1220097.
-        ActivityLog.create(amo.LOG.EXPERIMENT_SIGNED, file_)
```

Comment:
Technically they will eventually be auto-signed as normal, it will just take slightly longer. I intended to mention those changes to releng on top of the public blog post we did, but you're right, we need to go wider internally, I'll do that.